### PR TITLE
v1

### PR DESCRIPTION
**Pull request by edublancas**
----
:books: Documentation preview :books:: https://ploomber-doc--3.org.readthedocs.build/en/3/

### CODE REVIEW

**Comment by neelasha23** [comment]
I see this on the signup page :
Screenshot 2023-07-01 at 11 23 47 AM

File: doc/quickstart/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## 3. Wait for the deployment to finish
+
+First, you'll see the "Deployment logs", which show the progress of installing your dashboards's dependencies.
+
+After a minute or so, you'll see the "Application logs", which expose your dashboard as a web application. The first deployment takes about 2 minutes.
+
+![](../static/dashboards-logs.png)
+
+Once the deployment is ready you'll see this in the "Application logs":
```
Comment (neelasha23) [inline doc/quickstart/dashboard.md:21]: "Application logs" -> `Application logs`

File: doc/quickstart/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## 2. Fill out the form
+
+
+![](../static/dashboards-form.png)
+
+## 3. Wait for the deployment to finish
+
+First, you'll see the "Deployment logs", which show the progress of installing your dashboards's dependencies.
```
Comment (neelasha23) [inline doc/quickstart/dashboard.md:15]: "Deployment logs" -> `Deployment logs`

File: doc/quickstart/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+After a minute or so, you'll see the "Application logs", which expose your dashboard as a web application. The first deployment takes about 2 minutes.
```
Comment (neelasha23) [inline doc/quickstart/dashboard.md:17]: "Application logs" -> `Application logs`

File: doc/dashboards/customize.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Customize your dashboard
+
+To deploy a dashboard you need two files:
+
+1. A Jupyter notebook files (`.ipynb`)
```
Comment (neelasha23) [inline doc/dashboards/customize.md:5]: `files` -> `file`

File: doc/dashboards/customize.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+2. A dependencies file (`requirements.txt`)
```
Comment (neelasha23) [inline doc/dashboards/customize.md:6]: Can users provide `environment.yml` as well?

File: doc/dashboards/customize.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+## Voilà
+
+Currently, Ploomber Cloud supports [Voilà](https://voila.readthedocs.io/en/stable/) for deploying Jupyter notebooks as dashboards. For information on how to write Voilà dashboards, [please check the documentation](https://voila.readthedocs.io/en/stable/).
+
+## Dependencies
```
Comment (neelasha23) [inline doc/dashboards/customize.md:12]: We can add another section after this that links to the deploy dashboard guide

File: doc/quickstart/dashboard.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+Once the deployment is ready you'll see this in the "Application logs":
+
+
+```
+Deployment with ID : {some id} completed successfully.
+```
+
+Scroll up and click on "View Dashboard"
+
+![](../static/dashboards-view.png)
```
Comment (neelasha23) [inline doc/quickstart/dashboard.md:30]: Can this be a bigger screenshot to understand the placement of the button better?

File: doc/quickstart/signup.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# Sign up
+
+## 1. Enter email and password
+
+To create a free account, go to the [sign up page](https://HOST/register).
```
Comment (yafimvo) [inline doc/quickstart/signup.md:5]: as @neelasha23 mentioned, the signup page redirects to `https://host/register`

**Review by yafimvo — COMMENTED** [review]
@edublancas I think it looks good and easy to understand.

The one thing I would change is the style of the quick links on the homepage (if possible)

image

Suggestions:
1. Increase the gap between the headline ("Ploomber Cloud documentation") and the 2 boxes 
2. Make the width of each box 50% of the main layout so it would be wider and the boxes' styles would match each other.
3. Make the main layout wider

Screenshot 2023-07-01 at 18 14 27

File: doc/static/dashboards-form.png
Comment (tonykploomber) [inline doc/static/dashboards-form.png:1]: 3. Click on "Create"

File: doc/dashboards/customize.md (hunk 5 above)
Comment (edublancas) [reply doc/dashboards/customize.md:6]: no, we currently only support requirements.txt

**Comment by tonykploomber** [comment]
Snapshot text error

Screenshot 2023-07-05 at 9 52 54 AM

**Comment by edublancas** [comment]
@yafimvo https://github.com/ploomber/doc/issues/6 we're not writing the CSS, but using the defaults from the theme. I opened an issue

**Pull request merged**